systemtests/CMakeLists.txt

```diff
@@ -623,6 +623,7 @@ set(SYSTEM_TESTS
     virtualfull
     virtualfull-bscan
     volume-pruning
+    ai-duplicate-job-cancel
```

Comment:
If not already done, we should order the list of tests alphabetically.
Also, the testname does not match what is being tested.
How about 
**ai-consolidation-ignores-duplicate-job-checking**

Reply:
Testname changed to "ai-consolidate-ignore-duplicate-job". Also sorted the whole list alphabetically.